oak_containers_launcher/src/lib.rs

```diff
@@ -115,10 +118,38 @@ pub fn path_exists(s: &str) -> Result<std::path::PathBuf, String> {
     }
 }
 
+#[derive(Clone)]
+pub enum Channel {
+    Network(u16, Option<SocketAddr>),
```

Comment:
Can you comment on the u16? or in fact, perhaps use the struct version of the enum variant, so you can give it a name.

```suggestion
    Network{foo: u16, bar: Option<SocketAddr>},
```

Reply:
Yep, that's much cleaner, thanks.